src/components/RadioButtonGroup/RadioButton.tsx

```diff
@@ -0,0 +1,65 @@
+import { theme } from '../../theme'
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { RadioButtonGroupContext } from './BaseRadioButtonGroup'
+
+interface RadioButtonProps {
+  disabled: boolean
+  value: string
+}
+
+const RadioButtonVisual = styled.div<{ 'aria-checked'?: boolean; 'aria-disabled'?: boolean }>`
+  width: 17px;
+  min-width: 17px;
+  display: flex;
+  height: 17px;
+  min-height: 17px;
+  border-radius: 20px;
+  stroke-width: 1;
+  ${({ 'aria-disabled': $disabled }) => {
+    return $disabled
+      ? css`
+          background-color: ${theme.grey10};
+          border: 1px solid ${theme.grey30};
+        `
+      : ''
+  }}
+  ${({ 'aria-checked': $selected, 'aria-disabled': $disabled }) => {
+    const color = $disabled ? theme.lochivar30 : theme.lochivar100
+    return $selected
+      ? css`
+          border: 1px solid ${color};
+          &:after {
+            background: ${color};
+            width: 11px;
+            height: 11px;
+            display: flex;
+            margin: auto;
+            border-radius: 10px;
+            content: '';
+          }
+        `
+      : `border: 1px solid ${theme.grey50};`
+  }}
+`
+
+const RadioButton: React.FC<React.ComponentPropsWithoutRef<'div'> & RadioButtonProps> = ({
+  disabled,
+  value,
+  ...rest
+}) => (
+  <RadioButtonGroupContext.Consumer>
+    {({ onSelectValue, selectedValue }) => (
+      <RadioButtonVisual
+        {...rest}
+        aria-disabled={disabled}
+        aria-checked={value === selectedValue}
+        onClick={() => onSelectValue(value)}
+        role="radio"
+        {...(!disabled && { tabIndex: 0 })}
```

Comment:
we should omit these props from `React.ComponentPropsWithoutRef<'div'>` if we're always going to override them so consumers are not fooled into thinking that they can pass these in.

Reply:
Ok. Unfortunately unknown props are allowed by default, so we'll need a new [linter rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md) to make this work end-to-end. I'll PR it after some experimentation.

Reply:
we don't need a linter rule since we have TypeScript. we just need to replace
`React.ComponentPropsWithoutRef<'div'>` with 
`Omit<React.ComponentPropsWithoutRef<'div'>, 'aria-disabled' | 'aria-checked' | 'onClick' | 'role' | 'tabIndex'>`